kingpin/actors/aws/elb.py

```diff
@@ -228,3 +216,93 @@ def _execute(self):
             yield utils.tornado_sleep(3)
 
         raise gen.Return()
+
+
+class UseCert(ELBBaseActor):
+
+    """Find a server cert in IAM and use it for a specified ELB."""
+
+    all_options = {
+        'name': (str, REQUIRED, 'Name of the ELB'),
+        'region': (str, REQUIRED, 'AWS region name, like us-west-2'),
+        'cert_name': (str, REQUIRED, 'Unique IAM certificate name, or ARN'),
+    }
+
+    # Local copy of {cert_name: cert_arn} dictionary
+    cert_arn_cache = {}
```

Comment:
Ok -- so we don't really do this anywhere else. I'd like to not do any caching like this because thats not a design we've followed anywhere else. Instead, a couple of extra API calls to search for the ARN are not a big deal, and they make the code simpler for now.